src/SFA.DAS.EmployerCommitmentsV2.Web/Views/CreateCohort/ConfirmProvider.cshtml

```diff
@@ -1,7 +1,57 @@
-﻿
+﻿@using SFA.DAS.EmployerCommitmentsV2.Web.Models.CreateCohort
+@model SFA.DAS.EmployerCommitmentsV2.Web.Models.CreateCohort.ConfirmProviderViewModel
+
 @{
     ViewData["Title"] = "ConfirmProvider";
 }
 
-<h1>ConfirmProvider</h1>
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <partial name="_ValidationSummary"/>
+
+        <h1 class="govuk-heading-xl">Confirm training provider</h1>
+        
+        <p class="govuk-body-l">UK Provider Reference Number <strong>'@Model.ProviderId'</strong> matches:</p>
+        
+        <h2 class="govuk-heading-l govuk-!-margin-bottom-0">@Model.ProviderName</h2>
+        <dl class="das-definition-list das-definition-list--inline  das-definition-list--muted govuk-!-margin-bottom-7">
+            <dt class="das-definition-list__title">UKPRN</dt>
+            <dd class="das-definition-list__definition">@Model.ProviderId</dd>
+        </dl>
+
+        <form method="post" asp-controller="CreateCohort" asp-action="ConfirmProvider">
+            <partial name="_ConfirmProviderHiddenPartial" model="@Model" />
```

Comment:
If this is only used on this page. Should it be in the the shared area

Reply:
Its following what the previous pages have done, i think its been done like this to allow us to consolidate this into one partial later on